modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/pagemem/PagesWriteThrottlePolicy.java

```diff
@@ -56,9 +92,10 @@ default void tryWakeupThrottledThreads() {
     void onFinishCheckpoint();
 
     /**
-     * @return {@code True} if throttling should be enabled, and {@code False} otherwise.
+     * Whether Checkpoint Buffer is currently in the danger zone.
+     *
+     * @return {@code true} if measures like throttling to protect Checkpoint Buffer should be applied,
+     * and {@code false} otherwise.
      */
-    default boolean shouldThrottle() {
-        return false;
-    }
+    boolean isCPBufferInDangerZone();
```

Comment:
The same rename suggestion.

Reply:
Renamed to `isCpBufferOverflowThresholdExceeded`